examples/ecs-firelens/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM openjdk:8-jdk-slim
+FROM openjdk:11
```

Comment:
could you remind me why jdk8 doesn't work? it just decide what jdk to use for your app. Could you add comment here if there's any constrain we should consider so that next developer can modify it in case needed?

Reply:
I think this is because your intellij using java 11 (correct me if I'm wrong) and you use java version 11 to compile. Then when we compiled the docker, the jdk is mismatched. Either your intellij change java version to 8 to meet docker, or made a change as you did.
Could you add a comment here mentioning make sure jdk matches your IDE compile version used for compiling Jar file

Reply:
Sure I can add the comment in the docker file itself.